controllers/cluster_controller_test.go

```diff
@@ -3158,7 +3158,9 @@ var _ = Describe("cluster_controller", func() {
 		JustBeforeEach(func() {
 			err := k8sClient.Create(context.TODO(), cluster)
 			Expect(err).NotTo(HaveOccurred())
-			Eventually(func() (int64, error) { return reloadCluster(cluster) }, 1).Should(Equal(int64(1)))
+			Eventually(func() (int64, error) {
+				return reloadCluster(cluster)
+			}, 5*time.Second).Should(Equal(int64(1)))
```

Comment:
The issue actually one happened here but I think the other statements are also fine and should make the test a little bit more robust.